lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts

```diff
@@ -6,7 +6,7 @@
 #include <lk2nd.dtsi>
 
 / {
-	qcom,msm-id = <QCOM_ID_MSM8917 0>, <308 0>, <309 0>;
+	qcom,msm-id = <QCOM_ID_MSM8917 0>, <QCOM_ID_MSM8217 0>, <QCOM_ID_MSM8617 0>;
```

Comment:
Maybe I've missed this before, but do we really need 3g versions of the soc here?

Reply:
They are defined in downstream i can try a boot without them.

Reply:
qcom always defines all "firmware" skus of the chip in the downstream soc dtsi, but I'd prefer to not have them unless needed

Reply:
It seems it works fine without it i will check ugglite also.